.github/workflows/releases.yml

```diff
@@ -26,6 +26,22 @@ jobs:
           args: release --release-notes=CHANGELOG.md
         env:
           GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
+  cards:
+    needs: goreleaser
+    runs-on: ubuntu-latest
+    if: ${!contains(github.ref, "-")}
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      HUB: 'bin/hub api -HACCEPT:application/vnd.github.inertia-preview+json '
```

Comment:
This feels awkward as an environment variable. If you wanted to cut on repetition in the bash script, then I suggest moving this to the script itself, either as a variable or, better yet, a function:

```sh
api() { bin/hub api -H 'accept: application/vnd.github.inertia-preview+json' "$@"; }
```